src/service/extensions-impl.js

```diff
@@ -238,15 +241,19 @@ export class Extensions {
    * Returns the promise that will be resolved when the extension has been
    * loaded. If necessary, adds the extension script to the page.
    * @param {string} extensionId
+   * @param {string=} extensionVer
    * @param {boolean=} stubElement
    * @return {!Promise<!ExtensionDef>}
    */
-  loadExtension(extensionId, stubElement = true) {
+  loadExtension(extensionId,
+      extensionVer = DefaultExtensionVersion, stubElement = true) {
```

Comment:
Default assignment removed. What's the best way to verify that the extension version is valid?